.github/workflows/ci-build.yml

```diff
@@ -45,18 +45,14 @@ jobs:
       fail-fast: false
       matrix:
         ember-try-scenario:
-          - ember-lts-3.8
-          - ember-lts-3.12
           - ember-lts-3.16
           - ember-lts-3.20
           - ember-lts-3.24
```

Comment:
Whether we do it in this PR or not, we probably want to replace the dropped rows with LTS 3.28, 4.4, 4.12. Personally I would be okay with just supporting 3.28+, but I am not sure what additional consideration we have for a "foundational package" like this one.